Frontend/src/Components/Dashboard/GraphContainer.js

```diff
@@ -64,22 +105,59 @@ export default function GraphContainer({ queue, latestTime, ...props }) {
       // save graph in old position, if needed
       // if (graphTitles[idx] && graphTitles[idx] !== "Custom")
       //   onSave(graphTitles[idx], datasets);
+      if (name === "") {
+        updateNeededGraphMetadata(idx, null)
+        let titles = graphTitles
+        titles[idx] = ""
+        setGraphTitles(titles)
+        return
+      }
 
-      // duplicate the graph titles
-      const copy = graphTitles.slice();
+      if ((name !== "Custom" || name !== "")) {
```

Comment:
```suggestion
      if ((name !== "Custom" && name !== "")) {
```
If you meant to check for graphName not being "Custom" or empty